### PR TITLE
Add remaining configuration options from dcos-cassandra

### PR DESCRIPTION
**Pull request by mrbrowning**

### CODE REVIEW

**Comment by mohitsoni** [comment]
LGTM!

**Comment by mohitsoni** [comment]
retest this please

**Comment by mrbrowning** [comment]
@mesosphere-ci retest this please

File: frameworks/cassandra/universe/marathon.json.mustache [hunk 1]
```diff
@@ -44,6 +46,7 @@
     "TASKCFG_ALL_CASSANDRA_HEAP_SIZE_MB": "{{nodes.heap.size}}",
     "TASKCFG_ALL_CASSANDRA_HEAP_NEW_MB": "{{nodes.heap.new}}",
     "CASSANDRA_HEAP_GC": "{{nodes.heap.gc}}",
+    "PLACEMENT_CONSTRAINT": "{{nodes.placement_constraint}}",
```
Comment (nickbp) [inline frameworks/cassandra/universe/marathon.json.mustache]: This option appears to be missing in config.json

File: frameworks/cassandra/universe/marathon.json.mustache [hunk 2]
```diff
@@ -21,10 +21,12 @@
...
   {{/service.secret_name}}
   "env": {
-    "FRAMEWORK_NAME": "{{service.name}}",
+    "SERVICE_NAME": "{{service.name}}",
     "SERVICE_USER": "{{service.user}}",
-    "FRAMEWORK_PRINCIPAL": "{{service.principal}}",
+    "SERVICE_ROLE": "{{service.role}}",
+    "SERVICE_PRINCIPAL": "{{service.principal}}",
     "CASSANDRA_VERSION": "3.0.10",
+    "SERVICE_ZK_SERVERS": "{{service.zk_servers}}",
```
Comment (nickbp) [inline frameworks/cassandra/universe/marathon.json.mustache]: Also missing in config.json

**Pull request merged**